shared/mail/mail.go

```diff
@@ -234,7 +237,7 @@ func TestConnection(config *SMTPConfig) error {
 	return nil
 }
 
-func SendMailWithEmbeddedFilesUsingConfig(to, subject, htmlBody string, embeddedFiles map[string]io.Reader, config *SMTPConfig, enableComplianceFeatures bool, ccMail string) error {
+func SendMailWithEmbeddedFilesUsingConfig(to, subject, htmlBody string, embeddedFiles map[string]io.Reader, config *SMTPConfig, enableComplianceFeatures bool, messageID string, inReplyTo string, references string, ccMail string) error {
```

Comment:
I just have a small nit, why did we put the new params before `ccMail` in the parameter list? Also, do we think the parameter list is getting a bit long? We could create a type for them, just a suggestion

Reply:
There was no particular reason why I put the new parameters before ccMail.

I'm not sure how creating a type would work, but if there's a way of shortening the parameter list, I'm all for it.